ca/ca.go

```diff
@@ -741,20 +741,18 @@ func (h *Handle) issueBatchTo(dir string, batch mtc.Batch, empty bool) error {
 				return nil
 			}
 
-			// TODO add not_after to abridged assertion and proof
-			// https://github.com/davidben/merkle-tree-certs/pull/92
-			aa := ar.Assertion.Abridge()
-			buf, err := aa.MarshalBinary()
+			be := ar.Assertion.Abridge(ar.NotAfter)
```

Comment:
minor: instead of `Abridge(ts)` it might make sense to have a function like this:
```
func NewBatchEntry(a Assertion, notAfter time.Time) BatchEntry {
  ...
}